### PR TITLE
Validation rule standardize on applies

### PR DESCRIPTION
**Pull request by xichan-ms**
[AB#357520](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/357520)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6994)

### CODE REVIEW

File: docs/specs/error.yml [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-              "hubpage",
-              "toc"
+        "images": {
+            "rules": [
+                {
+                    "type": "ImageAltTextEqualsFilename",
+                    "message": "AltText {0} is the same as the image file name.",
+                    "code": "altText-equal-filename",
+                    "severity": "SUGGESTION",
+                    "contentTypes": [
```
Comment (yufeih) [inline docs/specs/error.yml:16]: What is the behavior if there is no `contentTypes`? It seems a lot of burden to force each rule to have this list of `contentTypes`. If there is a default set, we could remove this property for most of the tests using regex. The python script alters the formatting of YAML files (which may conflict with manual edits).

Comment (xichan-ms) [reply docs/specs/error.yml:16]: OK

**Comment by xichan-ms** [comment]
The diff is expected. @yufeih @OsmondJiang pls help to review.

**Review by yufeih — APPROVED** [review]

**Review by OsmondJiang — APPROVED** [review]

**Pull request merged**